README.md

```diff
@@ -195,6 +195,43 @@ Done with this data source? Remove it with the following command:
 DROP DATABASE my_pg;
 ```
 
+## Supported data sources
+
+| Source                 | Read | Write[^1] | Table Function | External Table | External Database | Supported Object Stores[^2] |
+| ---------------------- | :--: | :-------: | :------------: | :------------: | ----------------- | --------------------------- |
+| **Databases**          |  --  |           |       --       |       --       | --                |                             |
+| MySQL                  |  ✅  |   ✅✅    |       ✅       |       ✅       | ✅                | ➖                          |
```

Comment:
I'm not sure what these double emojis are for (or the 5 emojis in the supported object stores column).

Let's remove the "Supported Object Stores" column since I think that might cause more confusion than it's worth.

And for the COPY TO vs INSERT ones, I think we should stick with `* COPY TO support only` immediately below the table and put `*` next to the insert emojis for the file formats that only support COPY TO.